src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+:active, :hover, :focus {
```

Comment:
What class should these pseudoclasses belong to?

Reply:
Class .form__input is added to pseudoclasses (rows: 34 - 39)